examples/developer_guide/3_simple_cpp_stage/CMakeLists.txt

```diff
@@ -13,52 +13,59 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-list(APPEND CMAKE_MESSAGE_CONTEXT "simple_cpp_stage")
+list(APPEND CMAKE_MESSAGE_CONTEXT "3_simple_cpp_stage")
 
-set(Python3_FIND_VIRTUALENV "FIRST")
-set(Python3_FIND_STRATEGY "LOCATION")
-find_package(Python3 REQUIRED COMPONENTS Development Interpreter NumPy)
+cmake_minimum_required(VERSION 3.24 FATAL_ERROR)
 
-pybind11_add_module(morpheus_example MODULE "${CMAKE_CURRENT_SOURCE_DIR}/_lib/pass_thru.cpp")
+# Set the cache to be the same to allow for CCache to be used effectively
+set(MORPHEUS_CACHE_DIR "${CMAKE_SOURCE_DIR}/.cache" CACHE PATH "Directory to contain all CPM and CCache data")
+mark_as_advanced(MORPHEUS_CACHE_DIR)
 
-add_library(${PROJECT_NAME}::morpheus_example ALIAS morpheus_example)
+# Add the Conda environment to the prefix path and add the CMake files
+list(PREPEND CMAKE_PREFIX_PATH "$ENV{CONDA_PREFIX}")
+list(PREPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../external/utilities/cmake")
```

Comment:
Lets export any functions needed by the examples from the Morpheus config.cmake file so users dont need to download or know about the necessary utilities functions. They should all be wrapped and exported in the config file.